### ISSUE TITLE
Windows 8+

### ISSUE THREAD

**Issue by Shadowolf7** [openSUSE/imagewriter#9]
Windows version in 8 and above demands .raw format though openSUSE images are all distributed in .iso. Windows 8 and above has no means to use a workaround. 

I'm working on trying to find where in the code this behavior happens, but I'm inexperienced developer.

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
This is incorrect. Windows 10 and 11 are offered as ISOs, else VirtualBox would cease to support them.

**Comment by bittin** [comment]
There is a Windows 8.1 iso too from Microsoft, but it has not been updated since the first release of Windows 8.1 in August 2013 also Windows 8.1 goes EOL in February next year 2023
